src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
+$border-color: #f3f3f3;
+$product-card-width: 200px;
+$main-fonts: 'Roboto';
+$main-text-color: #060b35;
+$second-text-color: #616070;
+$button-color: #00acdc;
+$border-radius: 5px;
+
 body {
   margin: 0;
 }
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $product-card-width;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
+  align-items: center;
+  justify-content: center;
+
+  &__image {
+    background-image: url('../images/imac.jpeg');
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+    box-sizing: border-box;
+  }
+
+  &__content {
+    margin: 40px 16px 16px;
+    font-family: $main-fonts, sans-serif;
+  }
+}
+
+.product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-text-color;
+}
+
+.product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $second-text-color;
+  margin: 4px 0 0;
+}
+
+.product-rating {
+  margin: 16px 0 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-text-color;
+  }
+}
+
+.stars {
+  flex-direction: row;
+  display: flex;
+  gap: 4px;
+
+  &__star {
+    background-image: url('../images/star-active.svg');
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:last-of-type {
+      background-image: url('../images/star.svg');
+    }
+  }
+}
+
+.product-price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 24px 0 0;
+
+  &__text {
+    line-height: 18px;
+    font-weight: 400;
+    font-size: 12px;
+    color: $second-text-color;
+  }
+
+  &__value {
+    line-height: 18px;
+    font-weight: 700;
+    font-size: 16px;
+    color: $main-text-color;
+  }
+}
+
+.buy-button {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: $button-color;
+  text-decoration: none;
+  color: #ffff;
```

Comment:
The color value should be written in lowercase to maintain consistency as per the code style rules.